runtime/src/lib.rs

```diff
@@ -474,14 +479,15 @@ impl pallet_committee::Config for Runtime {
     type ProposalSubmissionPeriod = ProposalSubmissionPeriod;
     type VotingPeriod = VotingPeriod;
     type MinCouncilVotes = MinCouncilVotes;
-    // Using root as the admin origin for now
-    type ProposalSubmissionOrigin = frame_system::EnsureRoot<AccountId>;
-    type ProposalExecutionOrigin = frame_system::EnsureRoot<AccountId>;
+    // Using signed as the admin origin for now
+    type ProposalSubmissionOrigin = frame_system::EnsureSigned<AccountId>;
+    type ProposalExecutionOrigin = frame_system::EnsureSigned<AccountId>;
```

Comment:
#96

Reply:
See also #64.
I think this is fine for now